.gitmodules

```diff
@@ -48,3 +48,6 @@
 [submodule "tpls/Stim"]
 	path = tpls/Stim
 	url = https://github.com/quantumlib/Stim
+[submodule "tpls/aws-sdk-cpp"]
+	path = tpls/aws-sdk-cpp
+	url = https://github.com/aws/aws-sdk-cpp.git
```

Comment:
Corresponding license entry in https://github.com/NVIDIA/cuda-quantum/blob/main/NOTICE file should be added.